plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -168,18 +157,23 @@ public function test_img_sizes_is_equal_to_picture_source_sizes_for_picture_elem
 
 		$img_markup = apply_filters( 'the_content', $image );
 
-		// Apply picture element support.
-		$this->opt_in_to_picture_element();
-
-		$picture_markup = apply_filters( 'the_content', $image );
-
 		$img_processor = new WP_HTML_Tag_Processor( $img_markup );
 		$this->assertTrue( $img_processor->next_tag( array( 'tag_name' => 'IMG' ) ), 'There should be an IMG tag.' );
 		$img_sizes = $img_processor->get_attribute( 'sizes' );
+		$img_src   = $img_processor->get_attribute( 'src' );
 
+		$this->assertStringEndsWith( '.jpg', $img_src, 'Should return .jpg img src.' );
+
+		// Apply picture element support.
+		$this->opt_in_to_picture_element();
+
+		$picture_markup    = apply_filters( 'the_content', $image );
 		$picture_processor = new WP_HTML_Tag_Processor( $picture_markup );
 
 		$picture_processor->next_tag( array( 'tag_name' => 'IMG' ) );
+		$picture_img_src = $picture_processor->get_attribute( 'src' );
+		$this->assertStringEndsWith( '.jpg', $picture_img_src, 'Should return .jpg img src.' );
+		$this->assertSame( $img_src, $picture_img_src, 'Make sure IMG src is same.' );
```

Comment:
Originally, this test was written to confirm that when `picture` is enabled, that the `source` elements will have the same `sizes` attribute as the `img` element that it is wrapping, which is why the test method is named `test_img_sizes_is_equal_to_picture_source_sizes_for_picture_element()`.

Adding these assertions here expands the scope of this test so it's now testing for multiple different things. Ideally, it would be best to have a specific test that relates to the bug being fixed, https://github.com/WordPress/performance/issues/1316, rather than mixing the purpose of a test we already have. Otherwise, we should at least update the test name to be more descriptive of what's being tested.

Reply:
Gotcha. Sorry for mixup. I make sure to check the sizes for the IMG is same. I have removed the src check from the test in https://github.com/WordPress/performance/pull/1408/commits/8b2bd7a437d50a777b9809c3581b2b3cef7dea85